app/components/Footer/footer.js

```diff
@@ -0,0 +1,128 @@
+]
+const listItems_2 = column_2.map((raw) => (
+    <p>{raw}</p>
+));              
+
+const column_3 =[
+    'Бытовая химия',
+                'Косметика и гигиена',
+                'Товары для дома',
+                'Товары для детей и мам',
+                'Посуда'
+]
+const listItems_3 = column_3.map((raw) => (
+    <p>{raw}</p>
+));
+
+const column_5 = [
+'<b>+7 (777) 490-00-91</b>',
+                'время работы: 9:00-20:00',
+                '<b>opt.sultan@mail.ru</b> ',
+                'на связи в любое время'
+]
+const listItems_5 = column_5.map((raw) => (
+    <p>{raw}</p>
+));
+
+
+export default function Footer() {
+  return (
+    <div className={style.footer}>
+    <div className={style.footer_text}>
+    <div className={style.column}>
+    <img src="/logo_white.png" height="64px" alt="Логотип Султан" />
+        <div className={style.footer_logo_section} ib="sultan">
+            <ul className={style.list}>
+                {listItems_1}
+                <li>
+                <TextForm 
+                    text="Введите ваш email..." 
+                    type="email"
+                    onClick={() => alert("Email в разработке")} 
+                    style={{ width: "284px", height: "60px" }}
+                    style_button={{width: "40px", height: "40px"}}
+                    icon="/arrow.png"
+                />
+                </li>
+            </ul>
+        </div>
+    </div>
+
+        <div className={style.column}>
+            <div className={style.menu}>
+            <ul className={style.list}>
+            <h3 className={style.menu_title}>Меню сайта:</h3>
+                {listItems_2}
+            </ul>
+            </div>
+        </div>
+
+        <div className={style.column}>
+        <div className={style.categories}>
+            <ul className={style.list}>
+            <h3 className={style.categories_title}>Категории:</h3>
+                {listItems_3}
+            </ul>
+        </div>
+        </div>
+
+        <div className={style.column}>
+            <div className={style.price}>
+            <ul className={style.list}>
+            <h3 className={style.price_list_title}>Скачать прайс-лист:</h3>
+            <div className={style.button}>
+                <Button 
+                    text="Прайс лист" 
+                    onClick={() => alert("Кнопка в разработке")} 
+                    icon="/download.png"
+                />
+                </div>
+            <div className={style.connection_mas}>
+                <li className={style.no_bullet}>Связь в мессенджерах</li>
+                <img src="/tg.png" height="40" alt="Telegram"></img>
+                <img src="/watsap.png" height="40" alt="WhatsApp"></img>
+            </div>
+            </ul>
+        </div>
+        </div>
+
+        <div className={style.column} id="contacts">
+        <div className={style.contacts}>
+            <ul className={style.list}>
+            <h3 className={style.contact_title}>Контакты:</h3>
```

Comment:
```suggestion
            <h3 className={style.contacTitle}>Контакты:</h3>
```
лучше таким образом именовать классы.